pkg/apb/deprovision.go

```diff
@@ -24,19 +23,14 @@ func Deprovision(instance *ServiceInstance, log *logging.Logger) error {
 		return err
 	}
 
-	if err = client.PullImage(instance.Spec.Name); err != nil {
-		return err
-	}
-
```

Comment:
Talked to @eriknelson, he thought that was just an artifact of earlier work and is no longer necessary.